dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -442,7 +443,7 @@
 
 		adc1: adc@40012000 {
 			compatible = "st,stm32-adc";
-			reg = <0x40012000 0x400>;
+			reg = <0x40012000 0x50>;
```

Comment:
Is this really part of the same commit? I think it needs a justification.

Reply:
@mbolivar-nordic The ADC1 device currently holds the address space for ADC1, ADC2, ADC3. 

The address for all ADC's starts at 0x40012000 with a length of 0x400. The following is the addresses of the individual ADC's:
ADC1: 0x40012000 length of 0x50
ADC2: 0x40012100 length of 0x50
ADC3: 0x40012200 length of 0x50
Common: 0x40012300 length of 0xC (need to double check the length of this one)

Maybe someone that knows stm32 well can comment on how to handle it.

Reply:
OK, then yes, I think this definitely needs a justification and likely a separate commit. 

> Maybe someone that knows stm32 well can comment on how to handle it.

Let's wait to see what the STM32 maintainers say.

Reply:
I do agree with @mbolivar-nordic. Please split this in a dedicated commit.